contrib/mesos/pkg/executor/service/service.go

```diff
@@ -45,8 +44,8 @@ type KubeletExecutorServer struct {
 	SuicideTimeout    time.Duration
 	LaunchGracePeriod time.Duration
 
-	kletLock sync.Mutex // TODO(sttts): remove necessity to access the kubelet from the executor
-	klet     *kubelet.Kubelet
+	klet      *kubelet.Kubelet // once set, immutable
```

Comment:
please add the TODO again. This is still valid. All this synchronization should not be necessary if we had decoupled both components.